CONTRIBUTING.md

```diff
@@ -167,6 +167,15 @@ features. For more information visit the [Babel documentation site]
 - https://saucelabs.com/beta/dashboard/tests
 - http://developer.samsung.com/remotetestlab/rtlDeviceList.action#
 
+#### Notes
+
+The CSS and JS files that are generated during the build task are only used
```

Comment:
Trivial and this is widely used, but earlier in this file it's `JavaScript` and this is the first mention of `JS`. Probably should add `JavaScript (JS)` to the earlier mention.